README.md

```diff
@@ -145,11 +155,77 @@ This method programatically hides the widget after it's been opened.
     key: 'mono_public_key',
     onSuccess: ({code}) => console.log("code", code),
   });
-  
+
   connect.setup();
   connect.open();
   // this closes the widget 5seconds after it has been opened
   setTimeout(() => connect.close(), 5000)
 ```
 
-If you find any issue using this package please let us know by filing an issue right [here](https://github.com/withmono/connect.js/issues)
\ No newline at end of file
+### <a name="onEventCallback"></a> onEvent Callback
+
+The onEvent callback returns two paramters, [eventName](#eventName) a string containing the event name and [data](#dataObject) an object that contains event metadata.
+
+```js
+const connect = new Connect({
+  key: 'mono_public_key',
+  onSuccess: ({code}) => console.log("code", code),
+  onEvent: (eventName, data) => {
+    console.log(eventName)
+    console.log(data)
+  }
+});
+
+```
+
+#### <a name="eventName"></a> `eventName`
+
+Event names corespond to the `type` key returned by the raw event data. Possible options are in the table below.
+
+| Raw Event Type | eventName |
```

Comment:
We don't have to document the raw event type, just the eventName and data object properties of each event